doc/man3/BIO_should_retry.pod

```diff
@@ -111,6 +111,21 @@ that is they cannot retry after a partial read or write. This is usually
 worked around by only passing the relevant data to ASN1 functions when
 the entire structure can be read or written.
 
+=head1 RETURN VALUES
+
+BIO_should_read(), BIO_should_write(), BIO_should_io_special(), and
+BIO_should_retry() return either true or false based on the actual conditions
+of the B<BIO>.
+
+BIO_retry_type() returns a mask of the cause of a retry condition or false if
+there is no retry condition.
```

Comment:
One can wonder if it would be appropriate to reword the description. I mean it doesn't actually return a mask, but rather a union [determined by masking specific flags]...